finat/tensorfiniteelement.py

```diff
@@ -12,7 +12,7 @@
 
 class TensorFiniteElement(FiniteElementBase):
 
-    def __init__(self, element, shape):
+    def __init__(self, element, shape, transpose=False):
```

Comment:
To put it plainly, should the DoF ordering be XXXX YYYY ZZZZ or XYZ XYZ XYZ XYZ (example for a P1 tetrahedron). `transpose=False` means the latter, `transpose=True` means the former. For normal vector elements, we use the latter ordering, FEniCS uses the former (expect for coordinates where FEniCS goes our way).